packages/fiori-elements-writer/test/unit/validate.test.ts

```diff
@@ -26,11 +26,11 @@ describe('Validate', () => {
         const feApp: FioriElementsApp<LROPSettings> = {
             ...Object.assign(feBaseConfig('felrop1'), {
                 template: {
-                    type: TemplateType.Worklist,
+                    type: TemplateType.FormEntryObjectPage,
                     settings: {}
                 },
                 service: {
-                    version: OdataVersion.v4 // Worklist does not support v4
+                    version: OdataVersion.v2 // Form Entry Object Page does not support v2
```

Comment:
This is confusing. You are setting it to `v2` and you are saying it is not supported. Which one is it? 😉

Reply:
Because worklist now supports v4, I didn't want to remove the test so I changed it into a different negative scenario which is FEOP-v2. It is a negative test. ```'Invalid ODataVersion for template type'```